### DISCUSSION TITLE
Lesson 3: VM execution error

### DISCUSSION THREAD

**Discussion by bozinovicn** [smartcontractkit/full-blockchain-solidity-course-py#1495] [Q&A]
Hello, 

When I deploy the FundMe contract 'fund' button works perfectly but when I clik getPrice and getVersion it just says 'VM execution error'.
Does anyone know what should I do?

**Comment by cromewar** [comment]
Hello @bozinovicn 
Well this depends on how your code is managed, please give a look to #980 to reformulate for question.
On the meantime please share your code here ;)
